### PR TITLE
Add docs for full stack application support

### PR DESCRIPTION
**Pull request by juanjoman**
Hi! I'm adding docs for our full-stack new feature

### CODE REVIEW

File: source/includes/_08-deployment.md [hunk 1]
```diff
@@ -32,6 +32,16 @@ It will take a while, but you should have your project deployed to your cloud pr
 
 If you make changes to your code, you can run `boost deploy` again to update your project in the cloud.
 
+### Introducing booster full-stack applications
```
Comment (javiertoledo) [inline source/includes/_08-deployment.md]:
```suggestion
### Deployment of static pages and assets
```

File: source/includes/_08-deployment.md [hunk 2]
```diff
@@ -32,6 +32,16 @@ It will take a while, but you should have your project deployed to your cloud pr
...
+
+Booster now uploads your frontend application to the cloud too! Just follow these steps:
```
Comment (javiertoledo) [inline source/includes/_08-deployment.md]:
```suggestion
Booster can deploy your static assets to the cloud too. To do so, follow the next steps:
```

File: source/includes/_08-deployment.md [hunk 3]
```diff
@@ -32,6 +32,16 @@ It will take a while, but you should have your project deployed to your cloud pr
...
+
+- Create a `public` directory in the root of your booster project
```
Comment (javiertoledo) [inline source/includes/_08-deployment.md:39]: Right now what we're deploying is the folder `public/dist`. Check out this comment https://github.com/theam/booster/pull/87/files#r427215068

File: source/includes/_08-deployment.md [hunk 4]
```diff
@@ -32,6 +32,16 @@ It will take a while, but you should have your project deployed to your cloud pr
...
+- Put your frontend application inside
+
+And that's it! If you run `boost deploy`, both backend and frontend will be deployed and
+you should see a CloudFront URL in your console's output
```
Comment (javiertoledo) [inline source/includes/_08-deployment.md]:
```suggestion
And that's it! If you run `boost deploy`, both backend and frontend will be deployed and distributed via CDN. You'll find the public URL in your console after deploy finishes.

> Note: In the current version, Booster expects that your application builds all its static assets in the `public/dist` folder, and that's the folder that will be published as the webroot. In future versions, this folder will be configurable, and `public` will be deployed by default instead.
```

**Review by javiertoledo — CHANGES_REQUESTED** [review]
I added several suggestions. It's better if we write docs in a more "atemporal" way, in a month this won't be "new" anymore :trollface: 

I also changed the word `CloudFront` by CDN, because we aim for a full abstraction

**Review by NickSeagull — COMMENTED** [review]
Agree with @javiertoledo 's comments

File: source/includes/_08-deployment.md (hunk 3 above)
Comment (alvaroloes) [reply source/includes/_08-deployment.md:39]: I think we could change it to deploy `public`. I added a response to that comment telling this

File: source/includes/_08-deployment.md (hunk 4 above)
Comment (alvaroloes) [reply source/includes/_08-deployment.md]: Check: https://github.com/theam/booster/pull/87/files#r428587708

**Review by alvaroloes — APPROVED** [review]
Agree with Javi's comments (it is pending the `public` vs `public/dist` folder), other than that, LGTM

File: source/includes/_08-deployment.md (hunk 3 above)
Comment (juanjoman) [reply source/includes/_08-deployment.md:39]: I'm changing it to deploy the content of `/public`

**Comment by juanjoman** [comment]
Already managed by this task: https://agilemonkeys.atlassian.net/browse/BOOST-694

**Pull request closed without merging**